src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+import java.util.ArrayList;
+
+public class TaskList {
+
+    protected ArrayList<Task> tasks;
```

Comment:
Perhaps can use List interface instead of ArrayList class?
So, it can accept different kinds of implementation of List interface.

Reply:
Do you mean an interface TaskList extends List<Task>?

Reply:
I mean can consider declaring `ArrayList<Task> tasks` as `List<Task> tasks`.